src/cmd-buildextend-metal

```diff
@@ -185,6 +182,23 @@ if [ "${rootfs_type}" = "ext4verity" ]; then
     BLKSIZE="$(getconf PAGE_SIZE)"
 fi
 
+disk_args=()
+qemu_args=()
+# SecureExecution extra stuff
+if [[ $secure_execution -eq "1" ]]; then
+    disk_args+=("--with-secure-execution")
+    if [ -z "${genprotimgvm}" ]; then
+        qemu_args+=("-drive" "if=none,id=hostkey,format=raw,file=$hostkey,readonly=on" \
+                            "-device" "virtio-blk,serial=hostkey,drive=hostkey")
+    else
+        genprotimg_img="$(mktemp -p "${PWD}" secex-XXXXXX.img)"
+        qemu-img create -f raw "${genprotimg_img}" 512M
```

Comment:
This could also be spelled `truncate -s 512M "${genprotimg_img}"`.

Reply:
I guess a raw image is basically just a bundle of reserved disc space, but i think calling `qemu-img create` makes it more clear what we want to create.
I'm fine either way, but i've left it as is for now.